tgbot/botmanager/models.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+MAX_LENGTH_USERNAME = 32
+MAX_LENGTH_FIRST_NAME = 15
+MAX_LENGTH_LAST_NAME = 20
+MAX_LENGTH_PHONE_NUMBER = 12
+USERNAME_VERBOSE = 'Имя пользователя в Telegram'
+LAST_NAME_VERBOSE = 'Фамилия пользователя Telegram'
+PHONE_NUMBER_VERBOSE = 'Номер телефона пользователя Telegram'
+
+
+class TelegramUser(AbstractUser):
```

Comment:
Это не юзеры , это просто их анкеты
```suggestion
class TelegramUser(models.Model):
```